apps/schema/static/schema/components/shipments/transactions.yaml

```diff
@@ -5,6 +5,7 @@ get:
   parameters:
     - $ref: '../core/parameters.yaml#/page'
     - $ref: '../core/parameters.yaml#/page_size'
+    - $ref: '../core/parameters.yaml#/ordering'
```

Comment:
This lists the ordering fields as 'updated_at' instead of 'modified_at'

Reply:
Ah, it actually should be updated_at as that's what we use in Transmission. That caused me to find issues with the Shipment and Transaction ordering_fields good catch!